module/motion/HeadController/src/HeadController.hpp

```diff
@@ -49,6 +49,7 @@ namespace module::motion {
         Eigen::Vector2f currentAngles;
```

Comment:
I can't comment on the line, but just above here there's a load of `static constexpr const char*`'s that don't seem to be used anymore?